src/main/webapp/app/home/home.component.ts

```diff
@@ -60,26 +58,25 @@ export class HomeComponent implements OnInit, AfterViewInit {
     ) {}
 
     ngOnInit() {
-        this.profileService
-            .getProfileInfo()
-            .pipe(
-                filter(Boolean),
-                tap((info: ProfileInfo) => {
-                    if (!info.activeProfiles.includes('jira')) {
-                        // if the server is not connected to an external user management such as JIRA, we accept all valid username patterns
-                        this.usernameRegexPattern = /^[a-z0-9_-]{3,100}$/;
-                        this.signInMessage = 'home.pleaseSignIn';
-                        this.errorMesssageUsername = 'home.errors.usernameIncorrect';
-                        this.externalUserManagementActive = false;
-                    } else {
-                        this.externalUserManagementUrl = info.externalUserManagementURL;
-                        this.externalUserManagementName = info.externalUserManagementName;
+        this.profileService.getProfileInfo().subscribe((profileInfo) => {
+            if (profileInfo) {
+                if (profileInfo.activeProfiles.includes('jira')) {
+                    this.externalUserManagementUrl = profileInfo.externalUserManagementURL;
+                    this.externalUserManagementName = profileInfo.externalUserManagementName;
+                    if (profileInfo.allowedLdapUsernamePattern) {
+                        this.usernameRegexPattern = new RegExp(profileInfo.allowedLdapUsernamePattern);
                     }
-
-                    this.isRegistrationEnabled = info.registrationEnabled || false;
-                }),
-            )
-            .subscribe();
+                } else {
+                    // TODO: in the future we might also allow external user management for non jira profiles
+                    this.externalUserManagementActive = false;
+                }
+                this.accountName = profileInfo.accountName;
+                if (this.accountName === 'TUM') {
```

Comment:
An idea for a future PR might be to allow to customize this, so that other can also set specific warnings/error messages.(?)